src/OpenLoco/src/Map/Track/Track.cpp

```diff
@@ -68,7 +68,7 @@ namespace OpenLoco::World::Track
                 {
                     continue;
                 }
-                if (!(getGameState().roadObjectIdIsFlag7 & (1 << roadObjectId)))
+                if (!(getGameState().roadObjectIdIsTram & (1 << elRoad->roadObjectId())))
```

Comment:
not sure how i made this mistake!

Reply:
Does this change mean there will be a divergence in this function in our replay tests?

Reply:
Luckily this code was only hit in ui code and even then it doesn't change the ui result. (This is because we haven't done much road code compared to track code)

So in summary no change to replays.

Reply:
Confirmed no divergence after 10000 ticks on the LP save.